### DISCUSSION TITLE
Low sampling rate (100 Hz)

### DISCUSSION THREAD

**Discussion by lvanderlaat** [audacity/audacity#2612] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
When I load wav files of low sampling rate (100 Hz) with seismic data, cannot assign the correct rate. 

**Describe the solution you'd like**
Support for this kind of low sampling rate.

**Comment by SteveDaulton** [comment, marked as answer]
Testing with both Audacity 3.0.2 and Audacity 2.4.2 I see no problem importing a 100 Hz sample rate WAV file correctly.

To change the "track sample rate", "Project rate", or "Default sample rate" to a non-standard rate (such as 100 Hz) it is necessary to select "Other" in the sample rate dialog box and then type in the required numeric  value. See: https://manual.audacityteam.org/man/quality_preferences.html

Please note that although Audacity supports non-standard sample rates, your sound card / audio interface may not.
